locale/en_US/LC_MESSAGES/clr-installer.po

```diff
@@ -837,3 +837,35 @@ msgstr "Could not interrupt %s"
 #, c-format
 msgid "Creating %s"
 msgstr "Creating %s"
+
+#, c-format
+msgid "Cleaning disk %s"
+msgstr "Cleaning disk %s"
+
+msgid "This will degrade RAID: %s"
+msgstr "This will degrade RAID: %s"
+
+msgid "Remove partition from RAID %s"
+msgstr "Remove partition from RAID %s"
+
+msgid "The Raid: [%s] has parts from other disks: [%s]"
+msgstr "The Raid: [%s] has parts from other disks: [%s]"
+
+msgid "Remove physical volume:%s from volume group: %s"
+msgstr "Remove physical volume:%s from volume group: %s"
+
+msgid "The Volume group: %s has physical volumes from other disks: [%s]"
```

Comment:
```suggestion
msgid "Volume group: %s has physical volumes from other disks: [%s]"
```

Reply:
This is similar to `msgid "RAID %s: has parts from other disks: [%s]"`
perhaps?
msgid "%s %s: has parts from other disks: [%s]"

Reply:
Were you hinting at unifying them? I think they are a little different. I have done the above suggestion